index.js

```diff
@@ -49,12 +49,21 @@ function fastifyView (fastify, opts, next) {
         return
       }
 
-      lru.set(page, engine.compile(html, options))
+      try {
+        lru.set(page, engine.compile(html, options))
+      } catch (error) {
+        that.send(error)
+        return
+      }
```

Comment:
Can you split `lru.set()` and `engine.compile`? I would like not to catch errors from `lru.set()`.

Reply:
ok, in next push I'll do that change